src/fileio.cpp

```diff
@@ -37,23 +37,24 @@ extern std::string _config_file;
 extern std::string _highscore_file;
 
 static const char * const _subdirs[] = {
-	"",
-	"save" PATHSEP,
-	"save" PATHSEP "autosave" PATHSEP,
-	"scenario" PATHSEP,
-	"scenario" PATHSEP "heightmap" PATHSEP,
-	"gm" PATHSEP,
-	"data" PATHSEP,
-	"baseset" PATHSEP,
-	"newgrf" PATHSEP,
-	"lang" PATHSEP,
-	"ai" PATHSEP,
-	"ai" PATHSEP "library" PATHSEP,
-	"game" PATHSEP,
-	"game" PATHSEP "library" PATHSEP,
-	"screenshot" PATHSEP,
-	"social_integration" PATHSEP,
+	"",                                     ///< BASE_DIR
+	"save" PATHSEP,                         ///< SAVE_DIR
+	"save" PATHSEP "autosave" PATHSEP,      ///< AUTOSAVE_DIR
+	"scenario" PATHSEP,                     ///< SCENARIO_DIR
+	"scenario" PATHSEP "heightmap" PATHSEP, ///< HEIGHTMAP_DIR
+	"gm" PATHSEP,                           ///< OLD_GM_DIR
+	"data" PATHSEP,                         ///< OLD_DATA_DIR
+	"baseset" PATHSEP,                      ///< BASESET_DIR
+	"newgrf" PATHSEP,                       ///< NEWGRF_DIR
+	"lang" PATHSEP,                         ///< LANG_DIR
+	"ai" PATHSEP,                           ///< AI_DIR
+	"ai" PATHSEP "library" PATHSEP,         ///< AI_LIBRARY_DIR
+	"game" PATHSEP,                         ///< GAME_DIR
+	"game" PATHSEP "library" PATHSEP,       ///< GAME_LIBRARY_DIR
+	"screenshot" PATHSEP,                   ///< SCREENSHOT_DIR
+	"social_integration" PATHSEP,           ///< SOCIAL_INTEGRATION_DIR
 };
+// This should line up with the "Subdirectory" enum defined in fileio_type.h
```

Comment:
I think this change made more sense when this PR used a file chooser.  I'll split it out.

What's the new preferred comment alignment scheme?

Reply:
The new alignment is generally "don't try to align anything," instead of trying to be consistent and making PRs bigger when things need to be realigned, as in this case. 🙂

In other words:
- If the section is already consistent, follow the pattern if you can without changing existing lines
- If the section isn't consistent, only add a single space and don't try to align anything
- If you're making a new block, only add a single space

Reply:
Thanks.